query.go

```diff
@@ -95,6 +98,15 @@ func (q *Query) Where(stmt string, args ...interface{}) *Query {
 		fmt.Println("Warning: Query is setup to use raw SQL")
 		return q
 	}
+	if inRegex.MatchString(stmt) {
+		var inq []string
+		for i := 0; i < len(args); i++ {
+			inq = append(inq, "?")
+		}
+		qs := fmt.Sprintf("(%s)", strings.Join(inq, ","))
+		stmt = strings.Replace(stmt, "(?)", qs, 1)
+		stmt = strings.Replace(stmt, "(?)", qs, 1)
```

Comment:
Why are you doing this line twice?

Reply:
@mclark4386 cause I liked it so much! ;) accidental vim at it's best.